### ISSUE TITLE
Implement a data tables

### ISSUE THREAD

**Issue by bcardarella** [ElixirMostWanted/elixirmostwanted.com#3]
## tables

wanteds:

* name: string
* purpose: text
* slug_id: string
* user_id: int (foreing key to the users table)
* completed: boolean

comments:

* wanted_id: int (foreign key to the wanteds table)
* user_id: int (foreign key to the users table)
* body: text

#Blocked by #2

**Comment by w0rd-driven** [comment]
Would completed_at as a timestamp be more useful to capture when it was completed? Is this scoped to just the schema or would it be helpful to generate the views as well?

**Comment by bcardarella** [comment]
yes, the timestamps should be there. I omitted them as I always presume they're there. I'll add to the spec

**Comment by bcardarella** [comment]
@w0rd-driven oh sorry, I misunderstood. You want to change `completed` to a timestamp. That works for me. Updating

**Comment by greven** [comment]
Hi,

Created a PR to this issue: #8.

**Comment by bcardarella** [comment]
Closed by #8
